ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.12.yaml

```diff
@@ -20,23 +16,23 @@ images:
 - dockerfile_path: operator/Dockerfile
   from: base
   inputs:
-    stolostron_builder_go1.21-linux:
+    stolostron_builder_go1.22-linux:
       as:
-      - registry.ci.openshift.org/stolostron/builder:go1.21-linux
+      - registry.ci.openshift.org/stolostron/builder:go1.22-linux
   to: multicluster-global-hub-operator
 - dockerfile_path: manager/Dockerfile
   from: base
   inputs:
-    stolostron_builder_go1.21-linux:
+    stolostron_builder_go1.22-linux:
       as:
-      - registry.ci.openshift.org/stolostron/builder:go1.21-linux
+      - registry.ci.openshift.org/stolostron/builder:go1.22-linux
   to: multicluster-global-hub-manager
 - dockerfile_path: agent/Dockerfile
   from: base
   inputs:
-    stolostron_builder_go1.21-linux:
+    stolostron_builder_go1.22-linux:
       as:
-      - registry.ci.openshift.org/stolostron/builder:go1.21-linux
+      - registry.ci.openshift.org/stolostron/builder:go1.22-linux
```

Comment:
We also need to upgrade the main branch: https://github.com/openshift/release/blob/master/ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-main.yaml